Add post for Go OTel metric SDK reaching stable status (#3333)
open-telemetry/opentelemetry.io: content/en/blog/2023/otel-go-metrics-sdk-stable.md

---
title: First stable release of OTel Go metric SDK
linkTitle: Go Metric SDK is Stable
date: 2023-09-29
author: '[Fabrizio Ferri-Benedetti](https://github.com/theletterf/) (Splunk)'
cSpell:ignore: Benedetti Fabrizio Ferri
---

> The Go metric SDK has finally reached stable status with version 1.19.0 of

Reviewer: nit

```suggestion
The Go metrics SDK has finally reached stable status with version 1.19.0 of
```

> The Go metric SDK has finally reached stable status with version 1.19.0 of

Reviewer: Do we want to add a hyperlink to https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.19.0?

OpenTelemetry Go. Project stability guarantees now apply to the
[go.opentelemetry.io/otel/sdk/metric](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/metric)

> package.

Reviewer: ```suggestion
module.
```

module (as there are more packages) 😉

> To get started with OTel Go metrics,
> [read the documentation](/docs/instrumentation/go/manual) or browse the
> [examples](https://github.com/open-telemetry/opentelemetry-go/tree/main/example)
> for demonstrations of different instrumentation scenarios covered by the
> OpenTelemetry Go Instrumentation.

Reviewer: I think we can simply refer to these instead
- `/docs/instrumentation/go/getting-started`
- `/docs/instrumentation/go/manual#metrics`

E.g.
```suggestion
To learn more check the [getting started](/docs/instrumentation/go/getting-started)
and [manual metrics](/docs/instrumentation/go/manual#metrics) documentation.
```

The getting started has reference to examples.
